.github/workflows/percy_snapshots.yml

```diff
@@ -40,6 +42,8 @@ jobs:
           node-version: 16.x
       - run: npm ci
       - run: ${{ inputs.build_script }}
+        env:
+          MAPBOX_API_KEY: ${{ secrets.MAPBOX_API_KEY }}
```

Comment:
does this naming work even though Storybook is expecting `REACT_APP_MAPBOX_API_KEY`?

Reply:
No I renamed it in storybook
```
env: (config) => ({
    ...config,
    REACT_APP_MAPBOX_API_KEY: process.env.MAPBOX_API_KEY,
  }),
```

Reply:
so would we need to maintain to keys locally then to have it work with Storybook and the test-site?

Reply:
It'll now check for either
```
env: (config) => ({
    ...config,
    REACT_APP_MAPBOX_API_KEY: process.env.MAPBOX_API_KEY || process.env.REACT_APP_MAPBOX_API_KEY,
  }),
```